pkgs/by-name/si/simpleBluez/package.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, dbus
+, fmt_9
+, lib
+}:
+let
+  pname = "simpleBluez";
+  version = "0.7.3";
+  src = fetchFromGitHub {
+    owner = "OpenBluetoothToolbox";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-CPBdPnBeQ0c3VjSX0Op6nCHF3w0MdXGULbk1aavr+LM=";
+  };
+in
+stdenv.mkDerivation {
+  inherit pname version src;
+
+  outputs = [
+    "out"
+    "dev"
+  ];
+
+  sourceRoot = "${src.name}/simplebluez";
+
+  cmakeFlags = [
+    "-DLIBFMT_LOCAL_PATH=${fmt_9.src}"
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    dbus
+  ];
+
+  meta = with lib; {
```

Comment:
No `meta.mainProgram` ?

Reply:
To my understanding this is just a library, not something you can execute.